pom.xml

```diff
@@ -57,6 +57,12 @@
             <artifactId>junit-platform-launcher</artifactId>
             <version>${junit-platform-version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-inline</artifactId>
+            <version>5.2.0</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
ToDo: Check if that isnecessary and delete if it is not

Reply:
necessary